packages/contracts-bedrock/scripts/libraries/DeployUtils.sol

```diff
@@ -18,6 +19,16 @@ library DeployUtils {
         require(_who.code.length > 0, string.concat("DeployUtils: no code at ", LibString.toHexStringChecksummed(_who)));
     }
 
+    function assertEIP1967ImplementationSet(address _proxy) internal view {
+        (bool success, bytes memory result) = _proxy.staticcall(abi.encodeWithSignature("implementation()"));
```

Comment:
We'll need to prank from the zero address for this call because of the `proxyCallIfNotAdmin` modifier. Let's all cast to the `Proxy` type and use a high level `Proxy(_proxy).implementation()` call instead of a low level call

Reply:
Latest implementation does this.